test/integration/targets/ansible-playbook-callbacks/all-callbacks.yml

```diff
@@ -0,0 +1,123 @@
+- hosts: localhost
+  gather_facts: false
+  vars_prompt:
+    name: vars_prompt_var
+    default: hamsandwich
+  handlers:
+    - name: handler1
+      debug:
+        msg: handler1
+
+    - debug:
+        msg: listen1
+      listen:
+        - listen1
+  roles:
+    - setup_remote_tmp_dir
```

Comment:
Just a note here, that any change to this role, will impact the output of the test.

There isn't a good solution here, other than calling this role independently of the test, and somehow passing the result into the next playbook.  I have ideas, but I'm not sure it's worth the added complexity.